resources/tentative/complex-static-html/src/big-dom-generator/params.js

```diff
@@ -0,0 +1,12 @@
+// Default seed for the random number generator.
```

Comment:
Instead of adding a comment like this, simply give the constant a more descriptive name.
e.g. DEFAULT_SEED_FOR_RANDOM_NUMBER_GENERATOR.

Reply:
Done.